packages/0x.js/src/utils/order_state_utils.ts

```diff
@@ -113,6 +117,13 @@ export class OrderStateUtils {
                 throw new Error(ExchangeContractErrs.InsufficientMakerFeeAllowance);
             }
         }
+        const minimumFillableTakerTokenAmountWithingNoRoundingErrorRange = signedOrder.takerTokenAmount
+                                                                           .times(1000)
+                                                                           .dividedBy(signedOrder.makerTokenAmount);
+        if (orderRelevantState.remainingFillableTakerTokenAmount
+            .lessThan(minimumFillableTakerTokenAmountWithingNoRoundingErrorRange)) {
+            throw new Error(ExchangeContractErrs.OrderFillRoundingError);
```

Comment:
I think we need a more specific error message name here. Perhaps: `CannotFillOrderWithoutCausingRoundingError`?

Reply:
Should it include some info about intervals?
Cause the above is not true. You can.